File: Front/ProEvents-App/src/app/models/Event.ts
import { Batch } from "./Batch";
import { LecturerEvent } from "./LecturerEvent";
import { SocialNetwork } from "./SocialNetwork";

export interface Event {
  id: number;
  local: string;
  eventDate?: Date;
  theme: string;
  qtyPeople: number;
  imageURL: string;
  phone: string;
  email: string;
  batchs: Batch[];
  socialNetworks: SocialNetwork[];
  lecturerEvents: LecturerEvent[];


}

File: Back/src/ProEvents.Domain/Event.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ProEvents.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Local { get; set; }

        public DateTime? EventDate { get; set; }

        [MaxLength(50)]
        public string Theme { get; set; }
        public int QtyPeople { get; set; }
        public string ImageURL { get; set; }

        public string Phone { get; set; }
        public string Email { get; set; }

        public IEnumerable<Batch> Batchs { get; set; }

        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<LecturerEvent> LecturerEvents { get; set; }


    }
}